### DISCUSSION TITLE
Ability to choose what shops sell

### DISCUSSION THREAD

**Discussion by astraylife1** [OpenRCT2/OpenRCT2#20798] [Feature requests]
The art deco stall from the expansion is the only stall that can sell more than one item. It would be useful if you could have a drop down menu for every stall and choose what the stall sells. You could have a T-shirt stall that sells t-shirts and maps or a burger stall that sells burgers and sodas. Or simply use the t-shirt stall to sell anything since it closely resembles and actual counter compared to most of the other stalls.

**Comment by ocalhoun6** [comment]
Maybe as a cheat.

Is it possible for a plugin to do this? I could see this as a feature of a 'shop/stall editor' plugin.
